regression/strings/smoke_tests/performance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+import time
+from subprocess import check_output
+
+git_output=check_output(["git", "show", "HEAD"])
+commit=git_output.split('\n')[0]
+
+commit_id=time.strftime("%Y_%m_%d__%H_%M_%S")+"__"+commit[7:]
+
+process = check_output(["grep", "^Runtime\ decision\ procedure", "-R"])
+
+file_name='performance_'+commit_id+'.out'
+print "writing to file", file_name
+f=open(file_name, 'w')
+
+for x in process.split('\n'):
+    f.write(x+'\n')
+
+f.close()
+
+print "drawing to file", file_name+".png"
+
+gnuplot_output = check_output(["gnuplot", "-e", "file='"+file_name+"'", "-e", "outputfile='"+file_name+".png'", "performance_draw.gp"])
+
+print gnuplot_output
```

Comment:
In that case, use `check_call` instead of `check_output`, which just permits `gnuplot` to write to stdout as usual.